sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/checks/BadClassNameCheck.kt

```diff
@@ -51,7 +51,7 @@ class BadClassNameCheck : AbstractCheck() {
         if (ktClass !is KtEnumEntry && !name.matches(formatRegex)) {
             context.reportIssue(
                 ktClass.nameIdentifier!!,
-                "Rename class \"$name\" to match the regular expression $format.",
+                """Rename class "$name" to match the regular expression: $format""",
```

Comment:
I'm not convinced. Why are we removing the `.` here? Because it could belong to the regex? In that case I would suggest using `"[regex]"` instead. 
Also, I think there should be no colon (`:`). The regex belongs to `the regular expression`, it's not something that provides a specification on something more generic (e.g. something like `...to match a valid format: regex` would be fine). Here, though, the regex is part of the regular sentence structure, so no `:` is needed.
```suggestion
                """Rename class "$name" to match the regular expression "$format".""",
```
EDIT: I saw that we already do this anyway in `VariableAndParameterNameCheck`.